docs/en/v0.4/user-guide/operations/configuration.md

```diff
@@ -61,8 +60,8 @@ greptime metasrv start --help
 - `--http-timeout <HTTP_TIMEOUT>`: HTTP request timeout in seconds.
 - `--selector <SELECTOR>`: You can refer [selector-type](/developer-guide/metasrv/selector#selector-type);
 - `--server-addr <SERVER_ADDR>`: The communication server address for frontend and datanode to connect to metasrv;
-- `--store-addr <STORE_ADDR>`: Etcd server addresses;
-- `--use-memory-store`: Use memory store instead of etcd;
+- `--store-addr <STORE_ADDR>`: Comma seperated etcd server addresses;
```

Comment:
What is the difference between `--store-addr` and `--server-addr`?

Reply:
`--store-addr` is the address of etcd, which stores our metadata